packages/blockchain-utils-validation/src/blockchains/__tests__/near.test.ts

```diff
@@ -0,0 +1,42 @@
+import { KeyPair } from 'near-api-js';
+import { validateLink } from '../near';
+import * as linking from "@ceramicnetwork/blockchain-utils-linking";
+import * as uint8arrays from 'uint8arrays';
+
+const did = 'did:3:bafysdfwefwe';
+const privateKey = 'ed25519:9hB3onqC56qBSHpHJaE6EyxKPyFxCxzRBkmjuVx6UqXwygvAmFbwnsLuZ2YHsYJqkPTCygVBwXpNzssvWvUySbd';
+const local_provider = KeyPair.fromString(privateKey);
+const chainRef = 'near-mainnet';
+
+class NearMockSigner {
```

Comment:
Does this mock the api used by near wallets?

Reply:
Yes, this is the raw wallet keypair, since the near-api-js module is mostly used against direct file store or localStorage. Idea here was to mimic the mnemonic setup in the other tests, and still product accurate signed results for NEAR wallets.